tests/integration/test_integration.py

```diff
@@ -870,6 +871,40 @@ def failure2(children):
             context.exception.args[0]
         )
 
+    def test_callback_dep_types(self):
```

Comment:
@alexcjohnson  do you want to migrate this with new style?

Reply:
not too hard to migrate -> https://github.com/plotly/dash/pull/761/commits/40525079c6f6af873ca68faec8ee2743de3c1c2e